HISTORY.rst

```diff
@@ -2,6 +2,13 @@
 
 Release History
 ===============
+0.1.14
```

Comment:
should not update this, how about moving this to another PR while bumping a newer version?